deploy/olm-catalog/multicluster-operators-subscription/manifests/apps.open-cluster-management.io_subscriptionreports_crd_v1alpha1.yaml

```diff
@@ -41,32 +43,73 @@ spec:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
             type: string
           metadata:
             type: object
           reportType:
-            description: ReportType is the name or identifier of the type of report
+            description: ReportType identifies the type of subscription report.
             enum:
             - Application
             - Cluster
             type: string
           resources:
-            description: Resources is an optional reference to the subscription resources
+            description: Resources is an optional reference to the subscription resources.
             items:
-              description: 'ObjectReference contains enough information to let you inspect or modify the referred object. --- New uses of this type are discouraged because of difficulty describing its usage when embedded in APIs.  1. Ignored fields.  It includes many fields which are not generally honored.  For instance, ResourceVersion and FieldPath are both very rarely valid in actual usage.  2. Invalid usage help.  It is impossible to add specific help for individual usage.  In most embedded usages, there are particular     restrictions like, "must refer only to types A and B" or "UID not honored" or "name must be restricted".     Those cannot be well described when embedded.  3. Inconsistent validation.  Because the usages are different, the validation rules are different by usage, which makes it hard for users to predict what will happen.  4. The fields are both imprecise and overly precise.  Kind is not a precise mapping to a URL. This can produce ambiguity     during interpretation and require a REST mapping.  In most cases, the dependency is on the group,resource tuple     and the version of the actual struct is irrelevant.  5. We cannot easily change it.  Because this type is embedded in many locations, updates to this type     will affect numerous schemas.  Don''t make new APIs embed an underspecified API type they do not control. Instead of using this type, create a locally provided and used type that is well-focused on your reference. For example, ServiceReferences for admission registration: https://github.com/kubernetes/api/blob/release-1.17/admissionregistration/v1/types.go#L533 .'
+              description: "ObjectReference contains enough information to let you
+                inspect or modify the referred object. --- New uses of this type are
```

Comment:
```suggestion
             You might have to deal with some of the following difficulties: 
               1. `Ignored fields`.  It includes many fields which are not
                generally valid.  For instance, ResourceVersion and FieldPath are
                both rarely valid in actual usage.  
                2. `Invalid usage help`. You cannot add specific help for individual usage.  In most embedded usages, you have particular restrictions, such as: `\"must refer only to types A and B\"` or ` \"UID not honored\"`  or ` \"name must
 be restricted\"`. You cannot describe these restrictions when they are embedded. 
                3. `Inconsistent validation`.  Because the usages are different, the
                validation rules are different by usage, making it difficult for you to predict what happens. 
                 4. The fields are both imprecise and overly precise.  `Kind` is not a precise mapping to a URL making the direction ambiguous, therefore, possibly requiring a REST mapping. In most cases, the dependency is on the `group,resource` setting and the version of the actual structure is irrelevant.  
                5. We cannot easily change the usage.  Because this type is embedded in many locations, updates to this type affects various schemas.  Therefore, avoid making new APIs and embedding an underspecified API type that they do not control. Instead, create a locally provided and used type that is focused on your reference. For example,  use`ServiceReferences` for admission registration by going to the following link: https://github.com/kubernetes/api/blob/release-1.17/admissionregistration/v1/types.go#L533
```